spec/helpers/collection_helpers.rb

```diff
@@ -29,42 +29,46 @@ def view_granule_results(col_name='15 Minute Stream Flow Data: USGS (FIFE)', fro
       # Execute with Ruby
       page.find("##{root} .panel-list-item", text: col_name).click
 
-      #item.click # This causes intermittent failures based on timing
+      # item.click # This causes intermittent failures based on timing
       wait_for_xhr
       wait_for_visualization_load
       # expect(page).to have_visible_granule_list
-    rescue => e
+    rescue StandardError => e
       Capybara::Screenshot.screenshot_and_save_page
-      puts "Visible overlay: #{OverlayUtil::current_overlay_id(page)}"
+      puts "Visible overlay: #{OverlayUtil.current_overlay_id(page)}"
       raise e
     end
 
-    def leave_granule_results(to='collection-results')
+    def leave_granule_results(_to = 'collection-results')
       wait_for_xhr
       # expect(page).to have_visible_granule_list
-      page.find("#granule-list a.master-overlay-back").click
-      #find('#granule-list').click_link('Back to Collections')
+      page.find('#granule-list a.master-overlay-back').click
+      # find('#granule-list').click_link('Back to Collections')
       wait_for_xhr
       wait_for_visualization_unload
       # expect(page).to have_visible_overlay(to)
-    rescue => e
+    rescue StandardError => e
       Capybara::Screenshot.screenshot_and_save_page
-      puts "Visible overlay: #{OverlayUtil::current_overlay_id(page)}"
+      puts "Visible overlay: #{OverlayUtil.current_overlay_id(page)}"
       raise e
     end
 
-    def hook_granule_results(col_name='15 Minute Stream Flow Data: USGS (FIFE)', scope=:all, from='collection-results')
+    def hook_granule_results(col_name = '15 Minute Stream Flow Data: USGS (FIFE)', scope = :all, from = 'collection-results')
       before(scope) { view_granule_results(col_name, from) }
       after(scope) { leave_granule_results }
     end
 
-    def hook_granule_results_back(col_name='15 Minute Stream Flow Data: USGS (FIFE)')
+    def hook_granule_results_back(col_name = '15 Minute Stream Flow Data: USGS (FIFE)')
       before :all do
         view_granule_results(col_name)
         leave_granule_results
       end
     end
 
+    def granule_count
```

Comment:
I'd suggest being more specific with this methods name. `granule_count` is used all over the place for variables but we also display granule count in multiple places and we want to ensure when we use this method we know exactly what page/field its looking for -- e.g. `search_results_granule_count` or something.